### PR TITLE
add a wrapper around cv2.resize for mask with arbitrary number of channels

### PR DESCRIPTION
**Pull request by baibaidj**
to support processing mask with more than 3 channels. 
This wrapper function is adopted from albumentation.

### CODE REVIEW

File: mmcv/image/geometric.py [hunk 1]
```diff
@@ -91,6 +91,42 @@ def imresize(img,
         h_scale = size[1] / h
         return resized_img, w_scale, h_scale
 
+def resize(img, size, dst=None, interpolation=cv2.INTER_LINEAR):
+    resize_fn = _maybe_process_in_chunks(cv2.resize, size=size, dst = dst, interpolation=interpolation)
+    return resize_fn(img) 
+
+def _maybe_process_in_chunks(process_fn, **kwargs):
+    """
```
Comment (hellock) [inline mmcv/image/geometric.py:99]: Please add proper reference if these codes are borrowed from other repos.

Comment (baibaidj) [reply mmcv/image/geometric.py:99]: got you.

**Comment by hellock** [comment]
Could you fix the linting and building errors?

**Comment by ZwwWayne** [comment]
This PR is closed since the modification is too complex. cv2.resize support images with 1,3,4 channels.

**Comment by OpenMMLab-Assistant-007** [comment]
Hi
@baibaidj 
We are grateful for your efforts in helping improve this open-source project during your personal time. 
Welcome to join OpenMMLab Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request closed without merging**